GPU/Common/FramebufferCommon.cpp

```diff
@@ -737,3 +737,29 @@ void FramebufferManagerCommon::NotifyBlockTransferAfter(u32 dstBasePtr, int dstS
 		}
 	}
 }
+
+void FramebufferManagerCommon::SetRenderSize(VirtualFramebuffer *vfb) {
+	float renderWidthFactor = (float)PSP_CoreParameter().renderWidth / 480.0f;
+	float renderHeightFactor = (float)PSP_CoreParameter().renderHeight / 272.0f;
```

Comment:
It doesn't really matter, but these lines can probably be removed now from `Framebuffer.cpp` and `FramebufferDX9.cpp` right?

-[Unknown]